test/JDBC/expected/col_length-vu-cleanup.out

```diff
@@ -60,3 +60,42 @@ DROP FUNCTION IF EXISTS col_length_prepare_f14();
 DROP FUNCTION IF EXISTS col_length_prepare_f15();
 DROP FUNCTION IF EXISTS col_length_prepare_f16();
 GO
+
+-- Drop types
+DROP TYPE custom_char_10;
+DROP TYPE custom_varchar_20;
```

Comment:
It's better to add `IF EXISTS` clause and a `GO` after each statement.
```
DROP TYPE IF EXISTS custom_char_10;
GO
```
It would make sure that one statement failure doesn't stop other statements from execution.